### PR TITLE
Fixes #2296 - send patch for lost commits

### LINKED ISSUE #2296: Investigate lost commits after rebasing refactor branch
@zoepage said the current refactor branch (which is on staging) is missing commits, or had some bad rebase side effects.

Ola, can you point us to what is wrong, so we can document it here?

### PR DESCRIPTION
**Pull request by zoepage**
r? @miketaylr 

not 100% sure, I got everything. But should cover the bigger chunks.

### CODE REVIEW

File: webcompat/static/js/lib/issues.js [hunk 1]
```diff
@@ -253,19 +253,20 @@ issues.ImageUploadView = Backbone.View.extend({
       return;
     }
 
-    var inlineHelp = $("<span></span>", {
+    // what is wc-Form-helpInline???
```
Comment (miketaylr) [inline webcompat/static/js/lib/issues.js]: Heh, probably something we can remove in the future...

File: webcompat/static/js/lib/issues.js [hunk 2]
```diff
@@ -274,9 +275,10 @@ issues.ImageUploadView = Backbone.View.extend({
     this.inputMap[id].valid = true;
     $(this.inputMap[id].elm)
       .parents(".js-Form-group")
-      .removeClass("wc-Form-error js-form-error")
-      .addClass("wc-Form-noError js-no-error");
+      .removeClass("js-form-error")
+      .addClass("js-no-error");
 
+    // this can probably go as we do not have wc- styles anymore?
```
Comment (miketaylr) [inline webcompat/static/js/lib/issues.js]: Let's file a cleanup bug to address these comments post-deploy.

File: webcompat/templates/home-page/form.html [hunk 3]
```diff
@@ -69,7 +69,7 @@ <h1 class="headline-1">Report Site Issue</h1>
     </div>
     <div class="form-text form-element js-Form-group">
       {{ form.os(class_='form-field text-field') }}
-      <span class="form-input-validation check js-bug-form-os"></span>
+      <span class="form-input-validation check"></span>
```
Comment (miketaylr) [inline webcompat/templates/home-page/form.html]: We need this class, it was added here to fix a failing test: https://github.com/webcompat/webcompat.com/commit/d07e1bb5409635abc3cb07cdf234a7ac6af82422

File: webcompat/templates/home-page/form.html [hunk 4]
```diff
@@ -85,7 +85,7 @@ <h1 class="headline-1">Report Site Issue</h1>
         {{ subradio.label(class_='form-label label-radio') }}
       {% endfor %}
     </div>
-    <div class="form-upload js-image-upload js-Form-group loader js-image-loader">
+    <div class="form-upload js-image-upload js-Form-group">
```
Comment (miketaylr) [inline webcompat/templates/home-page/form.html]: I think this is wrong too, https://github.com/webcompat/webcompat.com/commit/1444cedd5c630c3c191348068645c29fb5a129e2

File: webcompat/templates/home-page/form.html (hunk 3 above)
Comment (miketaylr) [reply webcompat/templates/home-page/form.html]: (i'm adding this)

**Comment by zoepage** [comment]
🎉 thanks! LMK, if you have questions!

File: webcompat/templates/home-page/form.html (hunk 4 above)
Comment (miketaylr) [reply webcompat/templates/home-page/form.html]: (i'm fixing this)

**Comment by miketaylr** [comment]
OK, i removed those form.html bits, an we should be good to merge once Travis is green.

File: webcompat/static/js/lib/issues.js (hunk 2 above)
Comment (miketaylr) [reply webcompat/static/js/lib/issues.js]: Filed https://github.com/webcompat/webcompat.com/issues/2298.

**Pull request merged**